nautobot/extras/models/jobs.py

```diff
@@ -401,6 +401,7 @@ class JobResult(BaseModel, CustomFieldModel):
         default=JobResultStatusChoices.STATUS_PENDING,
     )
     data = models.JSONField(encoder=DjangoJSONEncoder, null=True, blank=True)
+    job_kwargs = models.JSONField(blank=True, default=dict, encoder=NautobotKombuJSONEncoder)
```

Comment:
This should be generating a migration. Please review: https://nautobot.readthedocs.io/en/stable/development/getting-started/#handling-migrations

Reply:
Thoughts on getting these kwargs over the API?

They should be read-only. We should not allow the editing of existing `JobResult`'s job_kwargs.

Reply:
> Thoughts on getting these kwargs over the API?
> 
> They should be read-only. We should not allow the editing of existing `JobResult`'s job_kwargs.

@bryanculver have added to API for job result

Reply:
![image](https://user-images.githubusercontent.com/23552972/177120928-d03be2ce-a580-46e1-adae-11c3d882afed.png)

Reply:
> This should be generating a migration. Please review: https://nautobot.readthedocs.io/en/stable/development/getting-started/#handling-migrations

done